platform/security/security-migratable/src/main/java/org/codice/ddf/security/migratable/impl/SecurityMigratable.java

```diff
@@ -52,20 +57,28 @@ public SecurityMigratable(String description, boolean isOptional,
 
     public MigrationMetadata export(Path exportPath) throws MigrationException {
         Collection<MigrationWarning> migrationWarnings = new ArrayList<>();
-        exportCrlFile(exportPath, migrationWarnings);
+        exportCrlFiles(exportPath, migrationWarnings);
         exportPdpDirectory(exportPath, migrationWarnings);
         return new MigrationMetadata(migrationWarnings);
     }
 
-    private void exportCrlFile(Path exportDirectory, Collection<MigrationWarning> migrationWarnings)
-            throws MigrationException {
-        LOGGER.debug("Exporting CRL from property [{}] in file [{}]...",
-                CRL_PROP_KEY,
-                FILE_CONTAINING_CRL_LOCATION.toString());
-        migratableUtil.copyFileFromJavaPropertyValue(FILE_CONTAINING_CRL_LOCATION,
-                CRL_PROP_KEY,
-                exportDirectory,
-                migrationWarnings);
+    private void exportCrlFiles(Path exportDirectory, Collection<MigrationWarning> migrationWarnings)
+        throws MigrationException {
+        for (Path propertiesPath : PROPERTIES_FILES) {
+            exportCrlFile(propertiesPath, exportDirectory, migrationWarnings);
+        }
+    }
+
+    private void exportCrlFile(Path propertiesPath, Path exportDirectory,
+            Collection<MigrationWarning> migrationWarnings) throws MigrationException {
+        LOGGER.debug("Exporting CRL from property [{}] in file [{}]...", CRL_PROP_KEY,
+                propertiesPath.toString());
+        String crlPathStr = migratableUtil.getJavaPropertyValue(propertiesPath, CRL_PROP_KEY);
+
+        if (StringUtils.isNotBlank(crlPathStr)) {
```

Comment:
Should we skip only if `crlPathStr == null` and return a warning (or throw an exception?) if the property value is blank?